pandas/core/generic.py

```diff
@@ -4898,7 +4897,12 @@ def describe_numeric_1d(series):
                           formatted_percentiles + ['max'])
             d = ([series.count(), series.mean(), series.std(), series.min()] +
                  [series.quantile(x) for x in percentiles] + [series.max()])
-            return pd.Series(d, index=stat_index, name=series.name)
+            if isinstance(self, ABCSeries):
+                return self._constructor(
+                    d, index=stat_index, name=series.name)
+            else:
+                return self._constructor_sliced(
```

Comment:
this doesn't make much sense.you would need more axes here and/or different meta-data. why are you doing this?